app/src/main/java/gitlin/kothub/ui/NotificationActivity.kt

```diff
@@ -1,18 +1,16 @@
 package gitlin.kothub.ui
 
-import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
-
-import org.jetbrains.anko.AnkoLogger
-import org.jetbrains.anko.debug
-import android.view.Gravity
-import android.widget.*
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.LinearLayoutManager
 import gitlin.kothub.R
 import gitlin.kothub.adapters.NotificationAdapter
 import gitlin.kothub.github.api.data.Notifications
-import kotlinx.android.synthetic.main.toolbar.*
+import gitlin.kothub.github.api.notifications
 import kotlinx.android.synthetic.main.activity_notifs.*
-import gitlin.kothub.github.api.*
+import kotlinx.android.synthetic.main.toolbar.*
+import org.jetbrains.anko.AnkoLogger
+import org.jetbrains.anko.debug
 
 class NotificationActivity : AppCompatActivity(), AnkoLogger {

```

Comment:
Extend the LifecycleAppCompatActivity, create the drawer and add it as a listener. (Need to pull master first)